jenkins/agent-base/Dockerfile.rhel8

```diff
@@ -0,0 +1,87 @@
+FROM quay.io/openshift/origin-jenkins-agent-base
+
+SHELL ["/bin/bash", "-o", "pipefail", "-c"]
+
+ENV SONAR_SCANNER_VERSION=3.1.0.1141 \
+    CNES_REPORT_VERSION=3.1.0 \
+    TAILOR_VERSION=1.3.0 \
+    GIT_LFS_VERSION=2.6.1 \
+    JAVA_HOME=/usr/lib/jvm/jre
+
+ARG APP_DNS
+ARG SNYK_DISTRIBUTION_URL
+
+RUN yum -y install openssl \
+    && yum clean all \
+    && rm -rf /var/cache/yum/*
+
+COPY ./import_certs.sh /usr/local/bin/import_certs.sh
+RUN import_certs.sh
+
+# Install Sonar Scanner.
+RUN cd /tmp \
+    && curl -LO https://repo1.maven.org/maven2/org/sonarsource/scanner/cli/sonar-scanner-cli/${SONAR_SCANNER_VERSION}/sonar-scanner-cli-${SONAR_SCANNER_VERSION}.zip \
+    && unzip sonar-scanner-cli-${SONAR_SCANNER_VERSION}.zip \
+    && mv sonar-scanner-${SONAR_SCANNER_VERSION} /usr/local/sonar-scanner-cli \
+    && rm -rf sonar-scanner-cli-${SONAR_SCANNER_VERSION}.zip \
+    && /usr/local/sonar-scanner-cli/bin/sonar-scanner --version
+ENV PATH=/usr/local/sonar-scanner-cli/bin:$PATH
+
+# Add sq cnes report jar.
+RUN cd /tmp \
+    && curl -L https://github.com/lequal/sonar-cnes-report/releases/download/${CNES_REPORT_VERSION}/sonar-cnes-report-${CNES_REPORT_VERSION}.jar -o cnesreport.jar \
+    && mkdir /usr/local/cnes \
+    && mv cnesreport.jar /usr/local/cnes/cnesreport.jar \
+    && chmod 777 /usr/local/cnes/cnesreport.jar
+
+# Install Tailor.
+RUN cd /tmp \
+	&& curl -LO https://github.com/opendevstack/tailor/releases/download/v${TAILOR_VERSION}/tailor-linux-amd64 \
+	&& mv tailor-linux-amd64 /usr/local/bin/tailor \
+	&& chmod a+x /usr/local/bin/tailor \
+	&& tailor version
+
+# Install GIT-LFS extension https://git-lfs.github.com/.
+RUN cd /tmp \
+    && mkdir -p /tmp/git-lfs \
+    && curl -LO https://github.com/git-lfs/git-lfs/releases/download/v${GIT_LFS_VERSION}/git-lfs-linux-amd64-v${GIT_LFS_VERSION}.tar.gz \
+    && tar -zxvf git-lfs-linux-amd64-v${GIT_LFS_VERSION}.tar.gz -C /tmp/git-lfs \
+    && bash /tmp/git-lfs/install.sh \
+    && git lfs version \
+    && rm -rf /tmp/git-lfs*
+
+# Optionally install snyk.
+RUN if [ -z $SNYK_DISTRIBUTION_URL ] ; then echo 'Skipping snyk installation!' ; else echo 'Installing snyk... getting binary from' $SNYK_DISTRIBUTION_URL \
+    && curl -L $SNYK_DISTRIBUTION_URL --output snyk \
+    && mv snyk /usr/local/bin \
+    && chmod +rwx /usr/local/bin/snyk \
+    && mkdir -p $HOME/.config/configstore/ \
+    && chmod -R g+rw $HOME/.config/configstore/ \
+    && echo 'Snyk CLI version:' \
+    && snyk --version \
+    && echo 'Snyk installation completed!'; \
+    fi
+
+# Set java proxy var.
+COPY set_java_proxy.sh /tmp/set_java_proxy.sh
+RUN . /tmp/set_java_proxy.sh && echo $JAVA_OPTS
+
+# Customize entrypoint.
+RUN mv /usr/local/bin/run-jnlp-client /usr/local/bin/openshift-run-jnlp-client
```

Comment:
Here the original executable is renamed to "openshift-run-jnlp-client", however in ods-run-jnlp-client.sh it is referenced by "ocp-run-jnlp-client". We need to use the same name in both places.

Reply:
:( Fixed in 9f121cc5ac651f220c27f459f82a9533c7af050a.